File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.js
function solution(s) {
    var ans = "";
    var words = s.split(" ");
    for (w of words) {
        for (let i = 0; i < w.length; i++) {
            if (i % 2 == 0)
                ans += w[i].toUpperCase();
            else
                ans += w[i].toLowerCase();
        }
        ans += ' ';
    }
    return ans.slice(0, s.length);
}

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    ans = ''
    words = s.upper().split(' ')
    
    for word in words:
        n = len(word)

        for i in range(n):
            # 짝수번째 알파벳 == 인덱스가 홀수인 알파벳
            if i % 2 == 1:
                ans += word[i].lower()
            # 홀수번째 알파벳 == 인덱스가 짝수인 알파벳
            else:
                ans += word[i]
        
        ans += ' '
        
    return ans[:-1]